Packs/SymantecDLP/Integrations/SymantecDLP/SymantecDLP.py

```diff
@@ -670,7 +670,7 @@ def fetch_incidents(client: Client, fetch_time: str, fetch_limit: int, last_run:
             )[0]), default=datetime_to_iso_format))
             incident_creation_time = incident_details.get('incident', {}).get('incidentCreationDate')
             incident: dict = {
-                'rawJSON': incident_details,
+                'rawJSON': json.dumps(incident_details),
```

Comment:
@guyfreund  I think we are doing 2 json.dumps. You can save the first one (before doing json.loads) and avoid this one.